libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -4,40 +4,76 @@
       autoFocus
       matInput
       type="search"
-      placeholder="Search for books to add to your reading list"
+      placeholder="{{ constants.CONSTANTS.SEARCH_BOOKS }}"
       formControlName="term"
     />
-    <button mat-icon-button matSuffix>
-      <mat-icon>search</mat-icon>
+    <button
+      mat-icon-button
+      matSuffix
+      aria-label="Search button"
+      [disabled]="!searchForm.valid"
+      data-testing="search-button"
+    >
+      <mat-icon aria-label="Search icon">{{
+        constants.BUTTONS.SEARCH_BUTTON
+      }}</mat-icon></button
+    ><button
+      mat-icon-button
+      matSuffix
+      aria-label="Reset Search"
+      data-testing="clear-button"
+      (click)="resetSearch()"
+    >
+      <mat-icon>{{ constants.BUTTONS.CLEAR_BUTTON }}</mat-icon>
```

Comment:
Added clear button so that users can easily clear out the search term at once. It also makes it easy to call the clearSearch action when the search button is disabled.
```suggestion
      <mat-icon>{{ constants.BUTTONS.CLEAR_BUTTON }}</mat-icon>
```